graphql/resolve/update_mutation_test.yaml

```diff
@@ -1457,48 +1457,6 @@
     message: |-
       failed to rewrite mutation payload because duplicate XID found: T1
 
-- name: "Duplicate XIDs in single mutation for Interface"
```

Comment:
Removed this test as it was wrong. As xid does not have interface=true flag. It is perfectly fine to have Student and Teacher with same xid.